src/xcode/ENA/ENA/Source/Scenes/Home/HomeInteractor.swift

```diff
@@ -212,11 +212,13 @@ extension HomeInteractor {
 			)
 			inactiveConfigurator?.activeAction = inActiveCellActionHandler
 		case .unknownOutdated:
-			inactiveConfigurator = HomeInactiveRiskCellConfigurator(
-				inactiveType: .outdatedResults,
-				previousRiskLevel: store.previousRiskLevel,
-				lastUpdateDate: dateLastExposureDetection
-			)
+			riskLevelConfigurator = HomeUnknown48hRiskCellConfigurator(
+				isLoading: isRequestRiskRunning,
+				lastUpdateDate: dateLastExposureDetection,
+				detectionInterval: detectionInterval,
+				detectionMode: detectionMode,
+				manualExposureDetectionState: riskProvider.manualExposureDetectionState,
+				previousRiskLevel: store.previousRiskLevel)
```

Comment:
`HomeUnknown48hRiskCellConfigurator ` completely replaced `HomeInactiveRiskCellConfigurator`?

if `HomeInactiveRiskCellConfigurator` isn't used anymore, you can delete it from the project

Reply:
I will check it. Thanks.

Reply:
```HomeInactiveRiskCellConfigurator``` is still used for ```inactive``` state.  I knew I saw it somewhere else. :) Which I good because I have a feeling that the story with risk calculation states will not end with this. :)